src/js/tracks/text-track.js

```diff
@@ -284,7 +68,7 @@ var loadTrack = function(src, track) {
   }));
 };
 
-var indexOf = function(searchElement, fromIndex) {
+const indexOf = function(searchElement, fromIndex) {
```

Comment:
we can actually remove `indexOf`now because we're assuming an `es5` environment now. The changes will need to be made from usage similar to `indexOf.call(this.activeCues_, active[i]` to `this.activeCues_.indexOf(active[i])`.